lib/jekyll/commands/new.rb

```diff
@@ -21,8 +22,11 @@ def init_with_program(prog)
         def process(args, options = {})
           raise ArgumentError, "You must specify a path." if args.empty?
 
+          new_blog_title = sanitized args
```

Comment:
`sanitized` is new and is used in only one place. I would prefer to see `new_blog_title` become a method that does the extraction on the args, so that I don't have to wonder what's going on here. Instead, I just see this (from further down):

`create_site new_blog_title(args), new_blog_path`

which is much clearer, IMHO.

`new_blog_title` then becomes:

```ruby
def new_blog_title(args)
    a = args.join(" ")
    a.tr("\\", "/").split("/").last
end
```